Source/WebCore/platform/graphics/FontCascadeFonts.h

```diff
@@ -148,6 +148,12 @@ inline const Font& FontCascadeFonts::primaryFont(const FontCascadeDescription& d
                 }
             }
         }
+
+        auto fontSizeAdjust = description.fontSizeAdjust();
+        if (m_cachedPrimaryFont && fontSizeAdjust.isFromFont) {
```

Comment:
Shouldn't `m_cachedPrimaryFont` be an `ASSERT` at this point?

Reply:
You are right. Is removing the m_cachedPrimaryFont check from the if condition OK? Or else, is putting an `ASSERT` expected?
```
ASSERT(m_cachedPrimaryFont);
auto fontSizeAdjust = description.fontSizeAdjust();
if (fontSizeAdjust.isFromFont) {
...
}
```

Reply:
Yes, this is what I was thinking of.

Reply:
Thanks for the confirmation.